### PR TITLE
Log Forwarding add topic on forwarding from project

### PR DESCRIPTION
**Pull request by mburke5678**
@sichvoge requested a topic on how to configure Log Forwarding to send application logs from a project.
https://issues.redhat.com/browse/OSDOCS-1573

### CODE REVIEW

**Comment by mburke5678** [comment]
@sichvoge Please take a quick look. If you approve of this approach, I will do a thorough edit and get a QE review. If this does not meet your expectations, please let me know what you expect.

**Comment by mburke5678** [comment]
@QiaolingTang would you be able to review this PR? Thank you very much.

**Comment by QiaolingTang** [comment]
Hi @mburke5678 , @anpingli is the owner of this feature, I think he can review this PR.

File: modules/cluster-logging-collector-log-forward-project.adoc [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+<8> Configuration for a pipeline to use the input to send project application logs to an external Fluentd instance.
+<9> The inputRefs is the `my-app-logs` input.
+<10 The `outputRefs` is the name of the output to use.
+<11> Optional: A label to add to the logs.
+<12> Configuration for a pipeline to send logs to other log aggregators.
+** Optional. Specify a name for the pipeline.
+** Specify which log types should be forwarded using that pipeline: `application,` `infrastructure`, or `audit`.
+** Specify the output to use with that pipeline for forwarding the logs.
+** Optional. Specify the `default` output to forward logs to the internal Elasticsearch instance.
+** Optional. One or more labels to add to the logs.
```
Comment (bobfuru) [inline modules/cluster-logging-collector-log-forward-project.adoc:73]: Should probably use the colon for `Optional:` in these lines, too.

Comment (bobfuru) [reply modules/cluster-logging-collector-log-forward-project.adoc:73]: Also wondering if each of these bullets should instead be an annotation reference? Although I get why you might not want to when there are already 12 of them. :)

**Review by bobfuru — COMMENTED** [review]
Left a few comments but otherwise LGTM!

File: modules/cluster-logging-collector-log-forward-project.adoc (hunk 1 above)
Comment (mburke5678) [reply modules/cluster-logging-collector-log-forward-project.adoc:73]: @bobfuru I called out each parameter in the previous pipeline. It seemed overkill to use call-outs again for the same parameters, But I wanted to describe what each parameter is doing. There are 4 other sections that describe how to forward logs in specific scenarios. I used this format in each. LMK if that doesn't make sense.

File: modules/cluster-logging-collector-log-forward-project.adoc [hunk 2]
```diff
@@ -0,0 +1,80 @@
+// Module included in the following assemblies:
+//
+// * logging/cluster-logging-external.adoc
+
+[id="cluster-logging-collector-log-forward-project_{context}"]
+= Forwarding application logs from specific projects
+
+You can use the Log forwarding API to send a copy of the application logs from specific projects to an external log aggregator instead of, or in addition to, the default Elasticsearch log store. The external log aggregator must be configured to accept the protocol you are using. You are responsible for configuring the external log aggregator to receive the logs from {product-title}.
+
+To configure forwarding application logs from a project, create a `ClusterLogForwarder` custom resource (CR) with at least one input from a project, one or more outputs to the external log aggregator that uses any supported forwarding protocol, and pipelines that use those inputs and outputs.
```
[the remark this answers is not in the document]
Comment (mburke5678) [reply modules/cluster-logging-collector-log-forward-project.adoc:10]: I tightened this up a bit. A similar paragraph appears in 4 other places. The other ones are tighter than this was. Wanted to keep the same structure throughout the assembly.

File: modules/cluster-logging-collector-log-forward-project.adoc (hunk 1 above)
Comment (bobfuru) [reply modules/cluster-logging-collector-log-forward-project.adoc]: SGTM! It does seem like overkill to do that again.

**Pull request merged**